### ISSUE TITLE
Implement PlacementExtension

### ISSUE THREAD

**Issue by 7evenk** [node-projects/web-component-designer#105]
- [x] Draw not rotated rectangle around object while moving
- [x] Show position >>> Jochen has already done it
![image](https://user-images.githubusercontent.com/5232626/175105654-8ded039c-f2e0-41e5-bf97-facea38309b6.png)

![image](https://user-images.githubusercontent.com/5232626/195175592-97615fb3-9b6f-472a-beea-e78a5e0ab962.png)

- [x] Apply Extension only once. Implement start method via PointerTool.
